challenge_5/scss/main.scss

```diff
@@ -0,0 +1,41 @@
+@import "../scss/variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  align-items: center;
+  background: $main-color;
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  justify-content: center;
+  position: relative;
+  width: 800px;
+
+
+  .ice-cream {
+    background: $secondary-color;
+    border-radius: 100px 100px 41px 41px;
+    height: 300px;
+    width: 200px;
+  }
+
+  .ice-cream-stick {
+    background: $tertiary-light-color;
+    border-radius: 0 0 20px 20px;
+    height: 100px;
+    width: 62px;
+
+    &::before {
+      background: $tertiary-dark-color;
+      content: " ";
+      height: 20px;
+      position: absolute;
+      width: 62px;
+    }
+  }
```

Comment:
I don't think these selectors need to be nested inside the container. They just work fine without the nesting, they don't need all the specificity.

Reply:
57fb396